dali/operators/generic/one_hot.h

```diff
@@ -74,12 +87,14 @@ class OneHot : public Operator<CPUBackend> {
   bool SetupImpl(std::vector<OutputDesc> &output_desc, const HostWorkspace &ws) override;
   void RunImpl(HostWorkspace &ws) override;
 
+  TensorLayout GetOutputLayout(const HostWorkspace &ws, int placement_axis) const;
 
   int num_classes_;
   int axis_;
   const DALIDataType output_type_;
   float on_value_;
   float off_value_;
+  const std::string layout_axis_name_;
```

Comment:
```suggestion
  char layout_axis_name_ = 0;
```
It's a single character.